packages/client/src/comp/states/BaseContainer.ts

```diff
@@ -48,20 +49,57 @@ interface InputData {
   value: string
 }
 
+/** Converts mm-dd-yyyy dates to mm/dd/yyyy */
+const normalizeDate = (date: string): string => date.replace(
+  /-/g,
+  '/',
+).replace(
+  /[^(0-9)|(/).]/g,
+  '',
+)
+
+/**
+ * Handles the default value of fields, returning a InputData
+ *
+ * @param id the InputId of the field
+ * @param queryValue The query value from AppHistory
+ * @param defaultValid The default valid state for this field when empty,
+ * for example optional fields have this value set to true
+ */
+const defaultInputData = (
+  id: InputId,
+  queryValue: string | undefined,
+  defaultValid: boolean,
+): InputData => {
+  if (queryValue) {
+    return {
+      valid: isInputValid(id, id === 'birthdate' ? normalizeDate(queryValue) : queryValue),
```

Comment:
I don't think we need to worry about `birthdate` not being normalized.  The URL will be supplied by organizers, which will test it first.  Normalizing voter input makes more sense.